framework/src/problems/EigenProblem.C

```diff
@@ -137,6 +138,29 @@ EigenProblem::setEigenproblemType(Moose::EigenProblemType eigen_problem_type)
   }
 }
 
+void
+EigenProblem::computeUserObjects(const ExecFlagType & type, const Moose::AuxGroup & group)
+{
+  FEProblemBase::computeUserObjects(type, group);
+
+  if (type == EXEC_INITIAL && group == Moose::PRE_IC && _auto_initialization)
+  {
```

Comment:
Why do we need to put `auto_initialization` into `computeUserObjects`?

Reply:
This needs to be before the application of ICs because if we have both auto initialization and ICs in the input, the later need to win, thus we cannot apply scaling after `projectSolution`.